src/main/java/at/hannibal2/skyhanni/features/inventory/ActiveBeaconEffect.kt

```diff
@@ -0,0 +1,49 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.events.GuiContainerEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.InventoryUtils
+import at.hannibal2.skyhanni.utils.InventoryUtils.getAllItems
+import at.hannibal2.skyhanni.utils.InventoryUtils.getUpperItems
+import at.hannibal2.skyhanni.utils.ItemUtils.getLore
+import at.hannibal2.skyhanni.utils.LorenzColor
+import at.hannibal2.skyhanni.utils.RegexUtils.matches
+import at.hannibal2.skyhanni.utils.RenderUtils.highlight
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraft.inventory.ContainerChest
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+@SkyHanniModule
+object ActiveBeaconEffect {
+
+    val config get() = SkyHanniMod.feature.inventory
+
+    private val patternGroup = RepoPattern.group("inventory.activebeaconeffect")
+    private val inventoryPattern by patternGroup.pattern(
+        "inventory",
+        "Profile Stat Upgrades"
+    )
+    private val slotPattern by patternGroup.pattern(
+        "slot",
```

Comment:
A little improvement for the key, Since a "slot" alone may lead to a collision down the line.
```suggestion
        "slot.active",
```